<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryUi</class>
 <widget class="QMainWindow" name="queryUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>query ui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>671</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" alignment="Qt::AlignTop">
      <widget class="QDateTimeEdit" name="fromTime">
       <property name="dateTime">
        <datetime>
         <hour>10</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2021</year>
         <month>11</month>
         <day>11</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2021</year>
         <month>11</month>
         <day>11</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDateTimeEdit" name="toTime">
       <property name="dateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>0</second>
         <year>2021</year>
         <month>11</month>
         <day>13</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2021</year>
         <month>11</month>
         <day>13</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushToExcell">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>export to excel</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>671</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tablewdg">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>codice fiscale</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ticket</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numero ingressi</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushToQuery">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>select query transaction</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
